src/extras/render-passes/render-pass-compose.js

```diff
@@ -229,6 +235,7 @@ class RenderPassCompose extends RenderPassShaderQuad {
         this.ssaoTextureId = scope.resolve('ssaoTexture');
         this.bloomIntensityId = scope.resolve('bloomIntensity');
         this.bcsId = scope.resolve('brightnessContrastSaturation');
+        this.tintId = scope.resolve('tint');
```

Comment:
Do we need to worry about name collisions? `tint` is quite generic.

Reply:
nope, they can be shared, as long as each rendering sets up the required value to it